src/main/groovy/com/calgaryscientific/gradle/VeracodeUploadFile.groovy

```diff
@@ -30,6 +30,26 @@ import groovy.transform.CompileStatic
 
 @CompileStatic
 class VeracodeUploadFile {
+    static void uploadFiles(VeracodeAPI veracodeAPI, File outputFile, Set<File> fileSet, Integer maxTries, Integer waitTime, Boolean delete) {
```

Comment:
do we have test that check if the file is deleted?

Reply:
Yes, for both app level and sandbox level task. For example: https://github.com/calgaryscientific/veracode-gradle-plugin/blob/dd14c94995fd673b40e1209344c1553b96f83f9b/src/test/groovy/com/calgaryscientific/gradle/VeracodeUploadFileTest.groovy#L64